packages/client/src/components/context/EmbedProvider.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+  import Provider from "./Provider.svelte"
+  import { onMount } from "svelte"
+
+  let data = {}
+
+  export function extractDomainFromUrl(url) {
+    const { hostname } = new URL(url)
+    const parts = hostname.split('.');
+    const tld = parts.slice(-2).join(".")
+    return tld
+  }
+
+  export function handleMessage(event) {
```

Comment:
NAB: we don't need to export these 2 functions since they're never consumed outside the component.